LOAN/RealEstateLoans/HomeMortgageDisclosureActCoveredMortgages.rdf

```diff
@@ -187,27 +187,27 @@
 		<fibo-fnd-utl-av:adaptedFrom>the Revised HMDA regulatory text.</fibo-fnd-utl-av:adaptedFrom>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-loan-ln-hmda;HowSubmitted">
+	<owl:Class rdf:about="&fibo-loan-rel-hmda;HowSubmitted">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-cls;Classifier"/>
 		<rdfs:label>how submitted</rdfs:label>
 		<skos:definition>category indicating whether the applicant or borrower submitted the application for the covered loan directly to the reporting financial institution</skos:definition>
 		<fibo-fnd-utl-av:adaptedFrom>2015 Revised HMDA regulation</fibo-fnd-utl-av:adaptedFrom>
 	</owl:Class>
 	
-	<owl:NamedIndividual rdf:about="&fibo-loan-ln-hmda;HowSubmitted-directly">
-		<rdf:type rdf:resource="&fibo-loan-ln-hmda;HowSubmitted"/>
+	<owl:NamedIndividual rdf:about="&fibo-loan-rel-hmda;HowSubmitted-directly">
+		<rdf:type rdf:resource="&fibo-loan-rel-hmda;HowSubmitted"/>
 	</owl:NamedIndividual>
 	
-	<owl:NamedIndividual rdf:about="&fibo-loan-ln-hmda;HowSubmitted-notDirectly">
-		<rdf:type rdf:resource="&fibo-loan-ln-hmda;HowSubmitted"/>
+	<owl:NamedIndividual rdf:about="&fibo-loan-rel-hmda;HowSubmitted-notDirectly">
+		<rdf:type rdf:resource="&fibo-loan-rel-hmda;HowSubmitted"/>
 	</owl:NamedIndividual>
 	
-	<owl:Class rdf:about="&fibo-loan-ln-hmda;Race">
+	<owl:Class rdf:about="&fibo-loan-rel-hmda;Race">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-cls;Classifier"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&skos;broaderTransitive"/>
-				<owl:someValuesFrom rdf:resource="&fibo-loan-ln-hmda;Race"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-rel-hmda;Race"/>
```

Comment:
don't like the use of SKOS here. Also the restriction would never permit a root 
More broadly it seems these categories (race and ethnicity) are applicable to all sorts of financial applications or actions (e.g. foreclosing) not only HMDA loan applications.

Reply:
@rivettp - again, I agree with you but spending the time on this ontology without SME support is not productive